node/nanoexpress/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "0.12.x"
```

Comment:
`0.12.x` what is this ? is it a `npm` **feature** ?

Reply:
This can install always latest version of `0.12`, for example, `0.12.999`

Reply:
This feature of `npm` is **awesome**, but I prefer using a strict version for this `PR`. 
I'll be great if you can add a new issue / PR to have this feature on each `node` based **frameworks**

Reply:
Done :)